### ISSUE TITLE
Example Macro don't work

### ISSUE THREAD

**Issue by brunocalado** [richardgaywood/foundry-penllawen-compendium-table-maker#6]
Hi. I tried this macro and I got an error.

```
var s = game.modules.get("penllawen-compendium-table-maker").api.getCompendiumSummariser();

await s.makeNewJournalNamed("This is my Journal");

await s.addJournalPageNamed("Gear")
  .addInputCompendium("swpf-core-rules.swpf-gear")
  .writeJournalPage();

await s.addJournalPageNamed("Skills")
  .addInputCompendium("swpf-core-rules.swpf-skills")
  .writeJournalPage();
```

![image](https://user-images.githubusercontent.com/4999783/223721295-ecacabca-7db9-469b-992f-cc1b66712f61.png)

**Comment by richardgaywood** [comment]
Bruno -- where did you get your input objects for this? The field that's throwing the `null` check shouldn't ever have null contents. If you make a new one through the UI, they come back initialised but empty. Did you make these items programmatically somehow?

**Comment by brunocalado** [comment]
> 

These items are from Savage Pathfinder official module.

They have some weirdnesse in them, yes

**Comment by richardgaywood** [comment]
I think this is fixed is v1.3.1
